components/tag-list/tag-list-item-mixin.js

```diff
@@ -58,11 +79,27 @@ export const TagListItemMixin = superclass => class extends superclass {
 					transition: none;
 				}
 			}
+			.tag-list-item-content + d2l-button-icon {
```

Comment:
This appears to be targeting the clear button.  Is it possible to apply a specific class to the button so we can avoid the sibling selector?  I am guessing we cannot simply target `d2l-button-icon` since their tag content might contain such an element?